src/Blocks/BlockBookShelf.h

```diff
@@ -0,0 +1,26 @@
+
+#pragma once
+
+#include "BlockHandler.h"
+
+
+
+
+class cBlockBookShelfHandler :
+	public cBlockHandler
+{
+	public:
+		cBlockBookShelfHandler(BLOCKTYPE a_BlockType) :
+			cBlockHandler(a_BlockType)
+		{
+		}
+
+		virtual cItems ConvertToPickups(NIBBLETYPE a_BlockMeta, cBlockEntity * a_BlockEntity, const cEntity * a_Digger, const cItem * a_Tool) override
+		{
+			if ((ItemCategory::IsTool(a_Tool->m_ItemType)) && (ToolHasSilkTouch(a_Tool)))

```

Comment:
(minor: redundant parentheses)

Reply:
I wanted to be respectful of CONTRIBUTING.md : `Add those extra parentheses to conditions, especially in C++`, if isn't needed i can delete them